addons/transitions/Transitions.gd

```diff
@@ -115,7 +130,8 @@ func _common_wait_for_fade(data:Array, fade_type, fade_seconds:float) -> void:
 	
 	if fade_type == FadeType.CrossFade or fade_type == FadeType.Instant:
 		var tween = Tween.new()
-		_root.add_child(tween)
+		_root.call_deferred("add_child", tween)
```

Comment:
It looks like this line has tabs instead of spaces. Other than this, your changes look good to me. I verified that everything still works transitioning to/from control-root scenes.

Reply:
I mean, every line has tabs instead of spaces on that file.